docs/operations/hotplug_volumes.md

```diff
@@ -0,0 +1,124 @@
+# Hotplug Volumes
+
+KubeVirt now supports hotplugging volumes into a running Virtual Machine Instance (VMI). The volume must be either a block volume or contain a disk image file just like any other regular volume. When a VM that has hotplugged volumes is rebooted, the hotplugged volumes will NOT be attached to the restarted VM unless the volumes are persisted.
+
+## Enabling hotplug volume support
+
+Hotplug volume support must be enabled in the feature gates to be supported. The
+[feature gates](./activating_feature_gates.md#how-to-activate-a-feature-gate)
+field in the KubeVirt CR must be expanded by adding the `HotplugVolumes` to it.
+
+## Virtctl support
+
+In order to hotplug a volume, you must first prepare a volume. This can be done by using a DataVolume (DV). In the example we will use a blank DV in order to add some extra storage to a running VMI
+
+```yaml
+apiVersion: cdi.kubevirt.io/v1beta1
+kind: DataVolume
+metadata:
+  name: example-volume-hotplug
+spec:
+  source:
+    blank: {}
+  pvc:
+    accessModes:
+      - ReadWriteOnce
+    resources:
+      requests:
+        storage: 5Gi
+```
+In this example we are using `ReadWriteOnce` accessMode, and the default FileSystem volume mode. Volume hotplugging supports all combinations of block volume mode and `ReadWriteMany`/`ReadWriteOnce`/`ReadOnlyMany` accessModes, if your storage supports the combination.
+
+### Addvolume
+
+Now lets assume we have started a VMI like the [Fedora VMI in examples](https://github.com/kubevirt/kubevirt/blob/main/examples/vmi-fedora.yaml) and the name of the VMI is 'vmi-fedora' we can add the above blank volume to this running VMI by using the 'addvolume' command  available with virtctl
+
+```bash
+$ virtctl addvolume vmi-fedora --volume-name=example-volume-hotplug
+```
+
+This will hotplug the volume into the running VMI, and set the serial of the disk to the volume name. In this example it is set to example-hotplug-volume.
+
+#### Serial
+You can change the serial of the disk by specifying the --serial parameter, for example:
+```bash
+$ virtctl addvolume vmi-fedora --volume-name=example-volume-hotplug --serial=1234567890
+```
+
+The serial will be used in the guest so you can identify the disk inside the by the serial. For instance in Fedora the disk by id will contain the serial
```

Comment:
I believe there is a word missing between "inside the <missing word> by the serial"

Reply:
yes, missing guest